addons/sale/tests/test_credit_limit.py

```diff
@@ -29,18 +30,36 @@ def setUpClass(cls):
         })
 
     def test_credit_limit_multicurrency(self):
-        self.partner.use_partner_credit_limit = True
-        self.partner.credit_limit = 50
-
-        self.product.lst_price = 45
-
-        order = self.empty_order
-        order.pricelist_id = self.buck_pricelist
+        self.partner_a.credit_limit = 50
 
-        self.empty_order.order_line = [
-            Command.create({
-                'product_id': self.product.id,
-                'product_uom_qty': 1.0,
-            }),
-        ]
+        order = self.env['sale.order'].create({
+            'partner_id': self.partner_a.id,
+            'pricelist_id': self.buck_pricelist.id,
+            'order_line': [
+                Command.create({
+                    'product_id': self.company_data['product_order_no'].id,
+                    'product_uom_qty': 1,
+                    'price_unit': 45.0,
+                    'tax_id': False,
+                })
+            ]
+        })
+        self.assertEqual(order.tax_totals['amount_total'] / order.currency_rate, 22.5)
```

Comment:
```suggestion
        self.assertEqual(order.amount_total / order.currency_rate, 22.5)
```
`tax_totals` is mostly used when checking values that are updated before saving. In this case, you can simply use `amount_total` (same below) :wink: